dll/cpl/appwiz/createlink.c

```diff
@@ -5,13 +5,14 @@
  * PROGRAMMER:      Gero Kuehn (reactos.filter@gkware.com)
  *                  Dmitry Chapyshev (lentind@yandex.ru)
  *                  Johannes Anderwald
+ *                  Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
  * UPDATE HISTORY:
  *      06-17-2004  Created
  */
 
 #include "appwiz.h"
-
 #include <tchar.h>
```

Comment:
This is unnecessary and can be removed

Reply:
In GCC build, #include <strsafe.h> requests <tchar.h> inclusion. It's a necessary to build.

Reply:
https://stackoverflow.com/questions/16852925/need-to-include-strsafe-h-after-tchar-h-error

Reply:
Sorry, I misgot. I will fix it.

Reply:
OK. Done.